main.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+    backend "s3" {
+        bucket  = "hackathon-pipeline-tfstates"
+        key     = "state/hackathon-pipeline-terraform.tfstate"
+        region  = "us-east-1"
+    }
+}
+
+provider "aws" {
+  region  = "us-east-1"
+}
+
+variable "aws_key_pair" {
+  default = "hackathon_pipeline"
+}
+
+variable "tool_name" {
+  default = "extra_server"
+}
```

Comment:
Is this variable used?

Reply:
yes, it is used in the module for jenkins